packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1553,4 +1225,60 @@ contract Deploy is Deployer {
         require(dac.bondSize() == daBondSize);
         require(dac.resolverRefundPercentage() == daResolverRefundPercentage);
     }
+
+    /// @notice Get the DeployInput struct to use for testing
+    function getDeployInput() public view returns (OPContractsManager.DeployInput memory) {
+        OutputRoot memory testOutputRoot = OutputRoot({
+            root: Hash.wrap(cfg.faultGameGenesisOutputRoot()),
+            l2BlockNumber: cfg.faultGameGenesisBlock()
+        });
+        IAnchorStateRegistry.StartingAnchorRoot[] memory startingAnchorRoots =
+            new IAnchorStateRegistry.StartingAnchorRoot[](5);
+        startingAnchorRoots[0] =
+            IAnchorStateRegistry.StartingAnchorRoot({ gameType: GameTypes.CANNON, outputRoot: testOutputRoot });
+        startingAnchorRoots[1] = IAnchorStateRegistry.StartingAnchorRoot({
+            gameType: GameTypes.PERMISSIONED_CANNON,
+            outputRoot: testOutputRoot
+        });
+        startingAnchorRoots[2] =
+            IAnchorStateRegistry.StartingAnchorRoot({ gameType: GameTypes.ASTERISC, outputRoot: testOutputRoot });
+        startingAnchorRoots[3] =
+            IAnchorStateRegistry.StartingAnchorRoot({ gameType: GameTypes.FAST, outputRoot: testOutputRoot });
+        startingAnchorRoots[4] =
+            IAnchorStateRegistry.StartingAnchorRoot({ gameType: GameTypes.ALPHABET, outputRoot: testOutputRoot });
+        string memory saltMixer = "salt mixer";
+        return OPContractsManager.DeployInput({
+            roles: OPContractsManager.Roles({
+                opChainProxyAdminOwner: msg.sender,
+                systemConfigOwner: cfg.finalSystemOwner(),
+                batcher: cfg.batchSenderAddress(),
+                unsafeBlockSigner: cfg.p2pSequencerAddress(),
+                proposer: cfg.l2OutputOracleProposer(),
+                challenger: cfg.l2OutputOracleChallenger()
+            }),
+            basefeeScalar: cfg.basefeeScalar(),
+            blobBasefeeScalar: cfg.blobbasefeeScalar(),
+            l2ChainId: cfg.l2ChainID(),
+            startingAnchorRoots: abi.encode(startingAnchorRoots),
+            saltMixer: saltMixer,
+            gasLimit: uint64(cfg.l2GenesisBlockGasLimit()),
+            disputeGameType: GameTypes.PERMISSIONED_CANNON,
+            disputeAbsolutePrestate: Claim.wrap(bytes32(cfg.faultGameAbsolutePrestate())),
+            disputeMaxGameDepth: cfg.faultGameMaxDepth(),
+            disputeSplitDepth: cfg.faultGameSplitDepth(),
+            disputeClockExtension: Duration.wrap(uint64(cfg.faultGameClockExtension())),
+            disputeMaxClockDuration: Duration.wrap(uint64(cfg.faultGameMaxClockDuration()))
+        });
+    }
+
+    function resetInitializedProxy(string memory _contractName) internal {
+        console.log("resetting initialized value on %s Proxy", _contractName);
+        address optimismPortalProxy = mustGetAddress(string.concat(_contractName, "Proxy"));
```

Comment:
Looks like we use this function for the systemConfig also, so we should change this `optimismPortalProxy` var name

Reply:
oh, good catch. Will fix.

Reply:
3d92be23a802